Modules/Core/Common/wrapping/itkImageToImageFilterA.wrap

```diff
@@ -22,6 +22,18 @@ itk_wrap_class("itk::ImageToImageFilter" POINTER)
         itk_wrap_template("${ITKM_VIF${d}}${ITKM_VID${d}}" "${ITKT_VIF${d}},${ITKT_VID${d}}")
         itk_wrap_template("${ITKM_VID${d}}${ITKM_VIF${d}}" "${ITKT_VID${d}},${ITKT_VIF${d}}")
       endif()
+      foreach(pixel_type "${WRAP_ITK_RGB};${WRAP_ITK_RGBA}")
```

Comment:
This might need to be done only for d=3 (RGB) and d=4 (RGBA).

Reply:
I am using `c` for the number of vector components in a pixel and `d` for the number of image dimensions.  Any number of image dimensions `d` can have RGB or RGBA values for their pixels.  Yes, the most natural value for RGB is a `c` of 3 and for RGBA is a `c` of 4.  However, in this case we are setting up casts to `VectorImage` which does not specify its value of `c` until runtime.  Unless I am misunderstanding your point, I think we don't need to change this.

Well, except that I will change the names of the `c` and `d` variables because they are not informative!

Reply:
You are right, d for image dimension is unrelated. I forgot about `VectorImage`s component count being determined at runtime. I withdraw the remark.